### ISSUE TITLE
No payment option

### ISSUE THREAD

**Issue by arshidkv12** [ppsreejith/react-native-paytm#28]
No payment option. How to solve?
![Image description](https://image.prntscr.com/image/SGgmEKQqQaCSiRbpEzSBiA.png)

**Comment by guru439** [comment]
you may be trying original paytm account with staging paytm api configuration. try using test credentials https://developer.paytm.com/docs/testing-integration/ this will help

**Comment by arshidkv12** [comment]
It is working. My problem is only showing wallet option only.

**Comment by guru439** [comment]
yes that is even I am looking for how to show all the other options it is only showing paytm wallet option.
